src/main/java/com/minecolonies/commands/ColoniesCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.command.ICommand;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import org.jetbrains.annotations.NotNull;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.jetbrains.annotations.NotNull'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)